### PR TITLE
Binder

### PR DESCRIPTION
**Pull request by apd-ypuranik**
## Fixes/Addresses/Summary/Motivation:

Sets up the Binder environment to allow anyone to run PRIMO examples without having to download/install software locally

## Changes proposed in this PR:
- Set up Binder configuration in .binder folder
- Add a short README document in the .binder folder

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

File: .binder/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Run PRIMO with Binder
+
+You can run PRIMO with [Binder](https://mybinder.org): a public cloud service that provides a temporary and shot-lived sandbox environment to run PRIMO without installing any software locally.
```
Comment (radhakrishnatg) [inline .binder/README.md:3]: *short*-lived

File: .binder/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+You can run PRIMO with [Binder](https://mybinder.org): a public cloud service that provides a temporary and shot-lived sandbox environment to run PRIMO without installing any software locally. 
+
+## Quickstart
+
+You can launch the Binder environment by clicking on the following badge: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/apd-ypuranik/primo-optimizer/binder)
+
+## Key Notes
+
+* Binder environments are automatically destroyed after a [few minutes of inactivity](https://mybinder.readthedocs.io/en/latest/about/user-guidelines.html#how-long-will-my-binder-session-last). To avoid lost work, please download the notebook file on your local machine periodically.
+* The Binder provided environment is **public and insecure**. Please do not use this environment to work with sensitive data.
```
Comment (radhakrishnatg) [inline .binder/README.md:12]: Binder-provided?

**Review by radhakrishnatg — CHANGES_REQUESTED** [review]
Minor typo

**Comment by apd-ypuranik** [comment]
Skipping pull request in favor of a new and cleaner one to be opened separately

**Pull request closed without merging**